app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @user = Post.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post = @post
+
```

Comment:
The task requires that we create a form to handle the creation of comments. It did not specifically ask that we create a view for it. So I put the form in the show view of the post controller. It will make it easy for the user to comment on a post easily without having to navigate to another page.